tests/devices/qubit_mixed/test_qubit_mixed_simulate.py

```diff
@@ -273,8 +274,225 @@ def test_broadcasting_with_extra_measurement_wires(self, mocker):
         assert spy.call_args_list[0].args == (qs, {0: 0, 2: 1})
 
 
-def test_finite_shot_not_implemented():
-    op = qml.RX(np.pi, [0])
-    qs = qml.tape.QuantumScript([op], [qml.expval(qml.Z(0))], shots=1000)
-    with pytest.raises(NotImplementedError):
-        simulate(qs)
+@flaky(max_runs=5, min_passes=1)
```

Comment:
I believe if you seed each test case, you wouldn't need `flaky`, and it wouldn't help either, since every execution should give you exactly the same result.

Reply:
Oh cool. I'll remove it later